lib/DDGC/Web/Controller/InstantAnswer.pm

```diff
@@ -88,7 +88,11 @@ sub iarepo_json :Chained('iarepo') :PathPart('json') :Args(0) {
 
     my %iah;
 
+IA:
     for my $ia (@x) {
+
+        next IA unless $ia->dev_milestone eq 'live'; # or ready?
```

Comment:
Why not select the dev milestone you need?

```perl
my @x = $c->d->rs('InstantAnswer')->search({
    repo => $repo,
    dev_milestone => 'live',
});
```

Also, would HashrefInflator + serialising to JSON directly work for this function?